packages/jupyter-ai/pyproject.toml

```diff
@@ -40,6 +40,14 @@ dependencies = [
 
 dynamic = ["version", "description", "authors", "urls", "keywords"]
 
+[project.entry-points."jupyter_ai.chat_handlers"]
+ask = "jupyter_ai:chat_handlers:AskChatHandler"
+clear = "jupyter_ai:chat_handlers:ClearChatHandler"
+default = "jupyter_ai:chat_handlers:DefaultChatHandler"
+generate = "jupyter_ai:chat_handlers:GenerateChatHandler"
+help = "jupyter_ai:chat_handlers:HelpChatHandler"
+learn = "jupyter_ai:chat_handlers:LearnChatHandler"
```

Comment:
Does this declaration work currently? This syntax doesn't look correct. See https://setuptools.pypa.io/en/latest/userguide/entry_point.html#entry-points-syntax, specifically the subsection "Nested objects".

Reply:
Seems like it should look like:

```
# note substitution of last colon with a period
ask = "jupyter_ai:chat_handlers.AskChatHandler"
```